### PR TITLE
Create 2022-11-17-monthly-meeting.md

### PR DESCRIPTION
**Pull request by mikofski**
meeting minutes for November

- [x] add meeting notes template
- [x] update attendees, chair, meeting date, etc.
- [x] record prior minutes approval vote
- [x] copy project updates
- [x] condense and summarize meeting discussion notes from agenda
- [x] meeting adjourned time

### CODE REVIEW

**Comment by mikofski** [comment]
@numfocus/disc-steering-committee November meeting notes are ready for approval. thanks!

File: committee/public-meeting-minutes/2022-11-17-monthly-meeting.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+* Maintenance tasks and contributor experience work on NumPy, SciPy, Matplotlib
+
+### Tania Allard 
+* DISC committee monthly meeting and slack conversations as needed 
+* Maintenance tasks on JupyterHub and Jupyter related projects
+* Community calls attendance: Jupyter, conda-forge
+
+### Tereza Iofciu
+* PyConDE&PyDataBer - call for chairs.. Democratising the conference 😀
+* PSF D&I work group: organising a open session call for the community (for January)
```
Comment (terezaif) [inline committee/public-meeting-minutes/2022-11-17-monthly-meeting.md:69]:
```suggestion
* PSF D&I work group: organising an open session call for the community (for January)
```

**Review by terezaif — APPROVED** [review]
LGTM

**Review by noatamir — APPROVED** [review]
LGTM

File: committee/public-meeting-minutes/2022-11-17-monthly-meeting.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* Minutes from prior meeting August 25, 2022:
+
+RESOLVED, that the DISC committee approve the minutes at https://github.com/numfocus/DISC-committee/blob/main/committee/public-meeting-minutes/2022-08-25-monthly-meeting.md as representing a true and accurate record of the August 25, 2022 meeting.
+
+Approved, 6-0-0
+
+* No quorum for September, so no meeting notes
+* Also no quorum for October, ditto
+
+## Committee reports for August
```
Comment (melissawm) [inline committee/public-meeting-minutes/2022-11-17-monthly-meeting.md:55]:
```suggestion
## Committee reports for October
```

I probably forgot to change this in the original notes!

**Review by melissawm — APPROVED** [review]
Thanks, @mikofski !

**Review by Ruv7 — APPROVED** [review]

File: committee/public-meeting-minutes/2022-11-17-monthly-meeting.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+- [invitee list brainstorm](https://docs.google.com/spreadsheets/d/158TAICTeqzqdonVwnh3aR-AMhD2Z3S9dBen-mL0Kujg/)
+- Arliss asked for review, comments, & suggestions
+- participants will likely end up combinatoin of invited folks and open call for participants
+- No criteria have been established, feel free to include people in spreadsheet (linked abouve)
+- Could piggyback with PyData Berlin (2023-04-17), PyCon US (2023-04-19), JupyCon (May), PyCon Italy (2032-05-25), EuroPy
+- Planning committee met with Gayle Ollington to discuss event organization and details
+- hot topics: deciding on date, location, venue
+- target max attendance at 50 people
+- next steps: define conference goals which will fascilitate who to invite, dates, and location
+- Noa to reach out to PyCon Ital organizers to see if rooms available for 50 folks
```
Comment (noatamir) [inline committee/public-meeting-minutes/2022-11-17-monthly-meeting.md:33]:
```suggestion
- Noa to reach out to PyCon Italy organizers to see if rooms available for 50 folks
```

Comment (noatamir) [reply committee/public-meeting-minutes/2022-11-17-monthly-meeting.md:33]: Done by @terezaif, so maybe we should amend this?!

**Review by noatamir — APPROVED** [review]

**Review by melissawm — APPROVED** [review]

**Comment by mikofski** [comment]
approved 6-0-0 in January meeting, see [notes](https://docs.google.com/document/d/13J66gq-sj-8hAdj44OKj2YmxitCZsDReC4lmtC5VBI0)

**Pull request merged**